docker-compose.yml

```diff
@@ -70,6 +70,9 @@ services:
       DATABASE_USER: ${DATABASE_USER}
       DATABASE_PASSWORD: ${DATABASE_PASSWORD}
       DATABASE_HOST: ${DATABASE_HOST}
+      REDCAP_CONNECTION_ENABLED: ${REDCAP_CONNECTION_ENABLED}
+      REDCAP_TOKEN: ${REDCAP_TOKEN}
+      REDCAP_URL: ${REDCAP_URL}
```

Comment:
This env var just gets assigned to `REDCAP_API_URL` [here](https://github.com/Australian-Genomics/CTRL/blob/d388bac125505f9383d2c6a0f0406f34bc12cc0b/config/initializers/environment.rb#L2). Maybe you should call it that for consistency?

Reply:
Thanks